test/src/main/java/org/apache/accumulo/test/functional/MergeIT.java

```diff
@@ -102,6 +124,306 @@ public void mergeSize() throws Exception {
     }
   }
 
+  @Test
+  public void noChopMergeTest() throws Exception {
```

Comment:
The test that were added are nice and the comments make them easier to understand.